src/lib/renderer.ts

```diff
@@ -57,10 +57,18 @@ export function generateHtml(
     <meta charset="utf-8" />
     <title>Getting Started with Vertex</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link
+   <link
       rel="stylesheet"
-      href="https://unpkg.com/@vertexvis/viewer@0.11.x/dist/viewer/viewer.css"
+      href="https://unpkg.com/@vertexvis/viewer@latest/dist/viewer/viewer.css"
```

Comment:
We pinned this on purpose, so people's demos don't break. Maybe it makes sense to have an CLI option to configure this.

Reply:
I'd still pin even with the CLI option, e.g. `@latest` is opt-in.

Reply:
I'm good with pinning it